evm/.gitignore

```diff
@@ -0,0 +1 @@
+.yarn
```

Comment:
when I do this sort of ad-hoc lab stuff, I tend to put it in `multichain-testing/scripts`.

Looks like this is a whole package. still... consider moving it under `multichain-testing`; maybe in `multichain-testing/test/ymax0`

Reply:
yes, one repo per deployable thingy is a good guideline.

OTOH, coordinating changes across repositories can be a pain, so sometimes putting stuff in agoric-sdk is convenient.
And we'll want to use it in `multichain-testing` or arrange for some other way to integration test the whole product.
For those reasons, we haven't moved Fast USDC nor ymax to their own repositories, yet.

I don't have a strong preference.